libraries/chain/wasm_interface.cpp

```diff
@@ -52,9 +59,7 @@ namespace eosio { namespace chain {
       wasm_validations::wasm_binary_validation validator(control, module);
       validator.validate();
 
-      const auto& pso = control.db().get<protocol_state_object>();
-
-      webassembly::eos_vm_runtime::validate_intrinsics( code, pso.whitelisted_intrinsics );
+      webassembly::eos_vm_runtime::validate( code, gpo.wasm_configuration, pso.whitelisted_intrinsics );
```

Comment:
eos_vm_runtime::validate is not correct here.  It will reject some contracts that WAVM would accept.

Reply:
Overlooked merge badness.